msal/src/Microsoft.Identity.Client/Internal/Requests/AuthenticationRequestParameters.cs

```diff
@@ -117,6 +118,30 @@ internal class AuthenticationRequestParameters
             return parameters;
         }
 
+#if DESKTOP || NETSTANDARD1_3
```

Comment:
Why is this needed?

Reply:
The client credential object is not available for the android/iOS platform so the compiler throws an error if it is not there.
#Resolved